doc/tutorials/menus.md

```diff
@@ -13,6 +13,8 @@ To add new custom menu entries, implement appropriate menu script/plugin functio
 
 You can use those argument objects to decide what elements to return. For example, in case of game menu, `GetGameMenuItemsArgs` contains `Games` field listing all currently selected games that are being used as a source for the game menu.
 
+When the function associated with the menu item is invoked, it will be passed an arguments object. For game menu actions, C# extensions will be passed [GameMenuItemActionArgs](xref:Playnite.SDK.Plugins.GameMenuItemActionArgs) and script extensions will be passed [ScriptGameMenuItemActionArgs](xref:Playnite.SDK.Plugins.ScriptGameMenuItemActionArgs). `GameMenuItemActionArgs` has a `Games` property to access the selected list of games. For main menu actions, C# extensions will be passed [MainMenuItemActionArgs](xref:Playnite.SDK.Plugins.MainMenuItemActionArgs) and script extensions will be passed [ScriptMainMenuItemActionArgs](xref:Playnite.SDK.Plugins.ScriptMainMenuItemActionArgs).

```

Comment:
I think this is correct, I don't fully understand the difference between `GameMenuItemActionArgs` and `ScriptGameMenuItemActionArgs`.

Reply:
One is passed when menu items are requested and the other one passed when menu item is invoked.

Reply:
No, what I was referring to was the script variant of the same classes.

ScriptMainMenuItemActionArgs and ScriptMainMenuItemActionArgs
vs
GetMainMenuItemsArgs and MainMenuItemActionArgs

Reply:
One set is for scripts and the other is for compiled plugins.

Reply:
Yeah, that's what I thought. Just making sure I understood it correctly. That's what's in the text I added to the docs here.